localstack/services/apigateway/provider.py

```diff
@@ -59,14 +62,37 @@
 from localstack.services.edge import ROUTER
 from localstack.services.moto import call_moto
 from localstack.services.plugins import ServiceLifecycleHook
-from localstack.utils.collections import PaginatedList, ensure_list, select_from_typed_dict
+from localstack.utils.collections import (
+    DelSafeDict,
+    PaginatedList,
+    ensure_list,
+    select_from_typed_dict,
+)
 from localstack.utils.json import parse_json_or_yaml
 from localstack.utils.strings import short_uid, str_to_bool, to_str
 from localstack.utils.time import now_utc
 
 LOG = logging.getLogger(__name__)
 
 
+def get_moto_backend(context: RequestContext) -> apigw_models.APIGatewayBackend:
+    return apigw_models.apigateway_backends[context.account_id][context.region]
+
+
+def remove_empty_attributes_from_rest_api(rest_api: RestApi, remove_tags=True):
```

Comment:
Might move this function at the bottom of the file, left it there while WIP and do not want to re-trigger the CI for cosmetic changes, will do before merge!